package.json

```diff
@@ -21,22 +21,22 @@
     "@azure/identity": "3.1.2",
     "@hapi/hapi": "^20.2.2",
     "@hapi/inert": "^7.0.0",
-    "applicationinsights": "^2.5.1",
+    "applicationinsights": "^0.19.0",
     "fuse.js": "^6.6.2",
     "hapi-pino": "^11.0.1",
+    "joi": "^17.6.0",
     "json2csv": "^5.0.7",
+    "node-fetch": "2.6.8",
     "pg": "^8.8.0",
     "pg-hstore": "2.3.4",
-    "sequelize": "7.0.0-alpha.9",
-    "joi": "^17.6.0",
-    "node-fetch": "2.6.8"
+    "sequelize": "7.0.0-alpha.9"
   },
   "devDependencies": {
```

Comment:
Whats the reasoning behind downgrading all the npm packages on this service?

Reply:
Also, the build are failing. Please take a look.

Reply:
These are automatic build changes due to npm audit fix. let me have a look.